lib/config/presets/codecommit/index.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { TextEncoder } from 'web-encoding';
+import { Platform, setPlatformApi } from '../../../modules/platform';
+import { PRESET_DEP_NOT_FOUND } from '../util';
+import * as codeCommit from '.';
+
+jest.unmock('../../../modules/platform');
+jest.mock('@aws-sdk/client-codecommit');
+jest.mock('@aws-sdk/client-iam');
+
+describe('config/presets/codecommit/index', () => {
+  let codeCommitPlatform: Platform;
+  let codeCommitClient: any;
+  let iamClient: any;
+
+  beforeEach(async () => {
+    setPlatformApi('codecommit');
+    const mod = require('@aws-sdk/client-codecommit');
```

Comment:
You also need to mock here again, if you import fresh.

You should better just mock `../../../modules/platform/codecommit` here

Reply:
i changed this already yesterday and commited,

Reply:
see https://github.com/PhilipAbed/renovate/blob/codecommit/lib/modules/platform/codecommit/index.spec.ts
should be outdated